### PR TITLE
Create a new Make target for building the OS with a dummy applet and without signing.

### PR DESCRIPTION
**Pull request by jiggoha**

### CODE REVIEW

**Comment by AlCutter** [comment]
I think this doesn't prevent embedding of the trusted applet, but only stops the OS from launching it.

Since the embedding of the applet is done by the compiler due to the "magic" `go:embed` comments in `main.go`, I think the only way to make it possible to enable/disable it would be to move those `taElf` and `taSigs` vars out into a pair of new `.go` files (e.g., `embed.go` and `noembed.go`) which have opposite build tags defined (e.g. we could use the `debug` tag to enable embedding, and have it disabled when `!debug`).

**Comment by AlCutter** [comment]
An intermediate quick fix could be to have your new Makefile target create an empty file somewhere and override APPLET_PATH so it's always the empty file which gets embedded - that'll be effectively the same as disabling the embed as the `load` check is on `len(taELF) > 0` I think.

Later we should probably do the debug build tag when we're nearer to being ready.

**Comment by jiggoha** [comment]
> An intermediate quick fix could be to have your new Makefile target create an empty file somewhere and override APPLET_PATH so it's always the empty file which gets embedded - that'll be effectively the same as disabling the embed as the `load` check is on `len(taELF) > 0` I think.
> 
> Later we should probably do the debug build tag when we're nearer to being ready.

Done, PTAL.

File: Makefile [hunk 1]
```diff
@@ -131,6 +138,11 @@ copy_applet:
 	cp ${APPLET_PATH}/trusted_applet.elf ${CURDIR}/trusted_os/assets/
 	cp ${APPLET_PATH}/trusted_applet.sig ${CURDIR}/trusted_os/assets/
 
+create_dummy_applet:
+	mkdir -p $(DIR)/assets
+	touch $(DIR)/assets/trusted_applet.elf
```
Comment (AlCutter) [inline Makefile:143]: I think `touch` will just update the `mtime` if the file already exists.
You'll probably have to either `-rm` these files first, or maybe do a `cp /dev/null ...` or `truncate` command instead?

Comment (jiggoha) [reply Makefile]: Good point, done.

**Review by AlCutter — APPROVED** [review]

**Pull request merged**